docker/README.md

```diff
@@ -0,0 +1,29 @@
+# Creating docker containers for snapcraft
+
+## Setup
+
+    sudo snap install docker
+
+Ensure networking works, in some cases `dockerd` might need to have something
+like `--dns=8.8.8.8` added to the command.
+
+## Creating containers
+
+Note that in the following commands, `--network host` is not strictly needed
+if a proper docker bridge is setup.
+
+Note that there are four separate Dockerfiles here, each one corresponding to
+their respective risk level:
+
+- **edge.Dockerfile**: Using the snap from edge
+- **beta.Dockerfile**: Using the snap from beta
+- **candidate.Dockerfile**: Using the snap from candidate
+- **stable.Dockerfile**: Using the snap from stable
+
+Build the docker image with:
+
+    docker build --no-cache -f <risk>.Dockerfile --label snapcore/snapcraft --tag snapcore/snapcraft:<risk> --network host .
```

Comment:
It would be cool to put this command inside a for loop with edge, beta, candidate and stable, to generate the four with a single pasted command. What do you think?

Reply:
Well, judging from how I've done it so far, the only time I run this command is when I'm poking at one specifically and want to rebuild just that one, not all. Otherwise I'd write a script to build them all, but since dockerhub does this automatically I don't see it being used. Thoughts?

Reply:
That makes sense. No loop then :)